serverless/serverless-installing-cli-linux-ibm-power.adoc

```diff
@@ -0,0 +1,7 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="installing-cli-linux-ibm-power"]
+= Installing the `kn` CLI for Linux on IBM Power Systems
+:context: installing-cli-linux-ibm-power
+
+include::modules/serverless-installing-cli-linux-ibm-power-tarball.adoc[leveloffset=+1]
+include::modules/serverless-installing-cli-linux-ibm-power-rpm.adoc[leveloffset=+1]
```

Comment:
Not sure why this is an assembly when there's already one for installing kn? It looks like you're including assemblies inside the installing-kn assembly also. Please revisit guidelines re how modules and assemblies work.